twoliter/src/tools.rs

```diff
@@ -0,0 +1,41 @@
+use anyhow::{Context, Result};
+use flate2::read::ZlibDecoder;
+use log::debug;
+use std::path::Path;
+use tar::Archive;
+use tempfile::TempDir;
+
+const TAR_ZIP_DATA: &[u8] = include_bytes!(concat!(env!("OUT_DIR"), "/tools.tar.zip"));
```

Comment:
nit: to signal to developers that they don't need to install `unzip` to be able to interact with this artifact if they're trying to debug it for some reason.

```suggestion
const TAR_GZ_DATA: &[u8] = include_bytes!(concat!(env!("OUT_DIR"), "/tools.tar.gz"));
```